docs/INDEX.md

```diff
@@ -34,6 +34,8 @@ npm install @types/marked # For TypeScript projects
 DOMPurify.sanitize(marked.parse(`<img src="x" onerror="alert('not happening')">`));
 ```
 
+**⚠️ Input: special ZERO WIDTH unicode characters (for example `\uFEFF`) might interfere with parsing. Some text editors add them at the start of the file (see: [#2139](https://github.com/markedjs/marked/issues/2139)).**
+
```

Comment:
Let’s add an example below (like we do for sanitizing)

Reply:
Amended the commit